client/scripts/controllers/server/threads.ts

```diff
@@ -178,6 +178,28 @@ class ThreadsController {
     });
   }
 
+  public async pin(args: {proposal: OffchainThread, }) {
+    return $.ajax({
+      url: `${app.serverUrl()}/pinThread`,
+      type: 'POST',
+      data: {
+        'jwt': app.user.jwt,
+        'thread_id': args.proposal.id,
+      },
+      success: (response) => {
+        const result = modelFromServer(response.result);
+        if (this._store.getByIdentifier(result.id)) {
+          this._store.remove(this._store.getByIdentifier(result.id));
+        }
+        this._store.add(result);
+        return result;
+      },
+      error: (err) => {
+        console.error(err);
```

Comment:
This seems like it should be a `notifyError` since the user is expecting some behavior which will not be reflected on the frontend? (we should also standardize on some error conventions down the line too, thought that's obviously out of scope...)